cloudlift/config/service_configuration.py

```diff
@@ -209,9 +207,8 @@ def _validate_changes(self, configuration):
         try:
             validate(configuration, schema)
         except ValidationError as validation_error:
-            log_err(validation_error.message + " in " +
+            raise UnrecoverableException(validation_error.message + " in " +
                     str(".".join(list(validation_error.relative_path))))
-            exit(0)
```

Comment:
Previously we exit with `0` for validation errors. I've made it consistent and simpler to return with exit `1`.